pennylane/plugins/expt_tensornet.py

```diff
@@ -84,19 +85,25 @@ class TensorNetwork(Device):
         "Identity": identity,
     }
 
+    backend = "numpy"
+    reshape = staticmethod(np.reshape)
+    array = staticmethod(np.array)
+    asarray = staticmethod(np.asarray)
+    real = staticmethod(np.real)
+    imag = staticmethod(np.imag)
+    abs = staticmethod(np.abs)
+
```

Comment:
Like the approach of only defining these separately for the original ``tensornet`` device and ``tensornet.tf``! Makes up for quite some code duplication